cabal-testsuite/src/Test/Cabal/Monad.hs

```diff
@@ -319,16 +337,14 @@ runTestM mode m = withSystemTempDirectory "cabal-testsuite" $ \tmp_dir -> do
                     testRelativeCurrentDir = ".",
                     testHavePackageDb = False,
                     testHaveRepo = False,
-                    testHaveSourceCopy = False,
                     testCabalInstallAsSetup = False,
-                    testCabalProjectFile = "cabal.project",
+                    testCabalProjectFile = Nothing,
```

Comment:
Mm, why is this change?

Reply:
This was just a drive-by thing, because ``--project-file=cabal.project` was passed to all tests.. even if the file didn't exist or do anything. It seemed better to only pass it when explicitly asked for by the user.

Reply:
Alright, it’s always better to be explicit, I guess. I’d be a little surprised if this wouldn’t break any tests…